Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Domain/Model/DbContextBase.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace MvvmSeed.Domain.Model
+{
+    public abstract class DbContextBase : DbContext
+    {
+        private enum DatabaseMode
```

Comment:
Yep, I want to keep it private for now, but related to the comment above, I might remove that enum completely.